extensions/amp-bind/0.1/bind-expression.js

```diff
@@ -51,7 +51,7 @@ const FUNCTION_WHITELIST = (function() {
      * @param {...?} items
      */
     /*eslint "no-unused-vars": 0*/
-    'copyAndSplice': function(array, start, deleteCount, items) {
+    'copyAndSplice': function copyAndSplice(array, start, deleteCount, items) {
```

Comment:
I put the BindArrays wrapper there for organizational purposes should we choose to add more functions. I think while we only have one or a few here having them all at a top level should be fine. :+1: